testtools/UART_interface/input.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set -e
+
+# basic setup of input script for python
+echo -e "help\r\n$SERIAL_TASK\r\n\r\nversion" | cat > input.txt
```

Comment:
>n$SERIAL_TASK [](start = 16, length = 13)

Can you quickly add a comment with types of tasks accepted by script?

Reply:
Where do you want me to put this comment?

Reply:
Comment added to yml file

Reply:
Two suggestion - 1) call it INPUT_SH_SERIAL_TASK so it is very clear that its piped to input.sh which triggers "Test commands", 2) add a TODO in the yml as well to suggest this is temporary

---
In reply to: [284944141](https://github.com/Azure/azure-iot-sdk-c/pull/1002#discussion_r284944141) [](ancestors = 284944141)